CHANGELOG.md

```diff
@@ -29,29 +29,43 @@ Types of changes (Stanzas):
 "Client Breaking" for breaking CLI commands and REST routes.
 "State Machine Breaking" for breaking the AppState
 
+
 Ref: https://keepachangelog.com/en/1.0.0/
 -->
 
 # Changelog
 
 ## [Unreleased]
+* (gaia) Bump [Liquidity](https://github.com/gravity-devs/liquidity) module to [v1.4.4](https://github.com/Gravity-Devs/liquidity/releases/tag/v1.4.4).
```

Comment:
```suggestion
* (gaia) Bump [Liquidity](https://github.com/gravity-devs/liquidity) module to [v1.4.6](https://github.com/Gravity-Devs/liquidity/releases/tag/v1.4.6).
```

Reply:
we haven't done this yet tho. maybe fine to leave and then let's merge in a PR for using 1.4.6 quickly